sdks/python/apache_beam/io/debezium.py

```diff
@@ -0,0 +1,159 @@
+  transform in a Beam Python pipeline.
+
+  There are several ways to setup cross-language Debezium transform.
+
+  * Option 1: use the default expansion service
+  * Option 2: specify a custom expansion service
+
+  See below for details regarding each of these options.
+
+  *Option 1: Use the default expansion service*
+
+  This is the recommended and easiest setup option for using Python Debezium
+  transform. This option requires following pre-requisites
+  before running the Beam pipeline.
+
+  * Install Java runtime in the computer from where the pipeline is constructed
+    and make sure that 'java' command is available.
+
+  In this option, Python SDK will either download (for released Beam version) or
+  build (when running from a Beam Git clone) a expansion service jar and use
+  that to expand transforms. Currently Debezium transform use the
+  'beam-sdks-java-io-expansion-service' jar for this purpose.
+
+  *Option 2: specify a custom expansion service*
+
+  In this option, you startup your own expansion service and provide that as
+  a parameter when using the transform provided in this module.
+
+  This option requires following pre-requisites before running the Beam
+  pipeline.
+
+  * Startup your own expansion service.
+  * Update your pipeline to provide the expansion service address when
+    initiating Debezium transform provided in this module.
+
+  Flink Users can use the built-in Expansion Service of the Flink Runner's
+  Job Server. If you start Flink's Job Server, the expansion service will be
+  started on port 8097. For a different address, please set the
+  expansion_service parameter.
+
+  **More information**
+
+  For more information regarding cross-language transforms see:
+  - https://beam.apache.org/roadmap/portability/
+
+  For more information specific to Flink runner see:
+  - https://beam.apache.org/documentation/runners/flink/
+"""
+
+# pytype: skip-file
+
+from enum import Enum
+from typing import List
+from typing import NamedTuple
+from typing import Optional
+
+from apache_beam.transforms.external import BeamJarExpansionService
+from apache_beam.transforms.external import ExternalTransform
+from apache_beam.transforms.external import NamedTupleBasedPayloadBuilder
+
+__all__ = ['ReadFromDebezium', 'DriverClassName']
+
+
+def default_io_expansion_service():
+  return BeamJarExpansionService('sdks:java:io:expansion-service:shadowJar')
+
+
+class DriverClassName(Enum):
+  MYSQL = 'MySQL'
+  POSTGRESQL = 'PostgreSQL'
+  ORACLE = 'Oracle'
+  DB2 = 'Db2'
+
+
+ReadFromDebeziumSchema = NamedTuple(
+    'ReadFromDebeziumSchema',
+    [('connector_class', str), ('username', str), ('password', str),
+     ('host', str), ('port', str), ('max_number_of_records', Optional[int]),
+     ('connection_properties', List[str])])
+
+
+class ReadFromDebezium(ExternalTransform):
+  """
+        An external PTransform which reads from Debezium and returns
+        a JSON string for each item in the specified database
```

Comment:
For Python it would be good to not return JSON strings - instead we should parse them and return the dictionaries. Thoughts?

Reply:
I agree, will do.